jupyter/datascience/ubi8-python-3.8/requirements.in

```diff
@@ -0,0 +1,18 @@
+# JupyterLab data science requirements
+scikit-learn==0.24.1
+kafka-python==2.0.2
+matplotlib==3.4.1
+pandas==1.2.4
+numpy==1.19.2
+boto3==1.17.11
+scipy==1.6.2
+
+# JupyterLab extensions
```

Comment:
Some extensions from the ODS CI framework are missing. See https://github.com/red-hat-data-services/ods-ci/blob/master/tests/Tests/500__jupyterhub/plugin-verification.robot#L17